src/index.ts

```diff
@@ -103,12 +106,32 @@ export const debugWrapper = (options: IOptions) => {
   }
 }
 
-export const clearDebug = () => {
-  if (child && child.process) {
-    execSync('kill -9 ' + child.process.pid);
-    child.process.kill();
+export const clearDebug = (exitCode?: number | undefined): void => {
+  if (child && child.process && child.process.pid > 0) {
+    const pid = child.process.pid;
+
+    try {
+      child.process.kill(pid, 0);
```

Comment:
It seems that the kill method does not need to pass PID parameters：

`subprocess.kill([signal])`

refs: https://nodejs.org/api/child_process.html#child_process_subprocess_kill_signal

Reply:
fixed